app/src/main/java/com/example/tasky/core/presentation/designsystem/theme/Type.kt

```diff
@@ -0,0 +1,87 @@
+package com.example.tasky.core.presentation.designsystem.theme
+
+import androidx.compose.material3.Typography
+import androidx.compose.ui.text.TextStyle
+import androidx.compose.ui.text.font.Font
+import androidx.compose.ui.text.font.FontFamily
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.unit.sp
+import com.example.tasky.R
+
+val Inter = FontFamily(
+    Font(
+        resId = R.font.inter_medium,
+        weight = FontWeight.Normal
+    ),Font(
+        resId = R.font.inter_regular,
+        weight = FontWeight.Normal
+    ),
+    Font(
+        resId = R.font.inter_semibold,
+        weight = FontWeight.SemiBold
+    ),
+    Font(
+        resId = R.font.inter_bold,
+        weight = FontWeight.Medium
+    ),
+)
+
+val Typography = Typography(
+    headlineSmall = TextStyle(
+        fontFamily = Inter,
+        fontWeight = FontWeight.Medium,
+        fontSize = 16.sp,
+        lineHeight = 24.sp
+    ),
+    headlineMedium = TextStyle(
+        fontFamily = Inter,
+        fontWeight = FontWeight.SemiBold,
+        fontSize = 20.sp,
+        lineHeight = 24.sp
+    ),
+    headlineLarge = TextStyle(
+        fontFamily = Inter,
+        fontWeight = FontWeight.Bold,
+        fontSize = 28.sp,
+        lineHeight = 30.sp
+    ),
+    bodySmall = TextStyle(
+        fontFamily = Inter,
+        fontWeight = FontWeight.Normal,
+        fontSize = 14.sp,
+        lineHeight = 20.sp
+    ),
+    bodyMedium = TextStyle(
+        fontFamily = Inter,
+        fontWeight = FontWeight.Normal,
+        fontSize = 16.sp,
+        lineHeight = 24.sp
+    ),
+    labelSmall = TextStyle(
+        fontFamily = Inter,
+        fontWeight = FontWeight.SemiBold,
+        fontSize = 14.sp,
+        lineHeight = 20.sp
+    ),
+    labelMedium = TextStyle(
+        fontFamily = Inter,
+        fontWeight = FontWeight.SemiBold,
+        fontSize = 16.sp,
+        lineHeight = 24.sp
+    )
+)
+
+object ExtendedTypography {
```

Comment:
Whenever a typography token (or any other design token) doesn’t map 1:1 to MaterialTheme, you can extend it with a property like this:
```
val Typography.headlineXSmall: TextStyle
    get() = ExtendedTypography.headlineXSmall

val Typography.labelXSmall: TextStyle
    get() = ExtendedTypography.labelXSmall
```
This avoids needing to call `ExtendedTypography.labelXSmall `or import `ExtendedTypography` everywhere. Instead, you can simply use:
`MaterialTheme.typography.labelXSmall`
You can dive a little bit more in [how to extend Material Theme properties here](https://developer.android.com/develop/ui/compose/designsystems/custom).

Reply:
I was thinking about something like extension, but didn't find anything. That will be helpful, thanks.